compiler/resolution/expandVarArgs.cpp

```diff
@@ -473,7 +473,8 @@ static void expandVarArgsBody(FnSymbol*      fn,
     needTuple = true;
   }
 
-  if (formal->intent == INTENT_CONST) {
+  if (formal->intent == INTENT_CONST || formal->intent == INTENT_CONST_IN ||
+      formal->intent == INTENT_BLANK) {
```

Comment:
I'm not sure about `INTENT_BLANK` here. The varargs should use the default intent of each component, and the default intent of `atomic` is `ref`.

Reply:
Sorry, I should have given you this context [from Brad](https://github.com/chapel-lang/chapel/issues/25858#issuecomment-2329880362):
> I feel like that'd be a worthwhile bug tradeoff to make in the short-term since it'd be overly strict rather than letting illegal stuff get through. (in addition, I also imagine that people don't rely on the default ref-ness of these types in a varargs context much)

Reply:
OK, in that case, please make a comment here that says it's a bug / TODO / workaround, and also note that in the PR message.